mgr/manager.go

```diff
@@ -223,12 +340,14 @@ func (m *Manager) Load() error {
 // if needed.
 func (m *Manager) CheckCA(spec *CertServiceManager) error {
 	if changed, err := spec.CA.Refresh(); err != nil {
+		metrics.ActionFailure.WithLabelValues(spec.Spec.Path, "CA").Inc()
```

Comment:
This should be a seperate metric- we haven't tried to execute the action, but the CA refresh failed.